examples/coap-server.c

```diff
@@ -703,6 +709,37 @@ verify_cn_callback(const char *cn,
   return 1;
 }
 
+static uint8_t *read_file_mem(const char* file, size_t *length) {
+  FILE *f = fopen(file, "r");
+  long size;
+  uint8_t *buf;
+
+  *length = 0;
+  if (!f)
+    return NULL;
+
+  fseek(f, 0, SEEK_END);
```

Comment:
This appears as a weird way of reading the file size. Why not using `fstat()` here as well?

Reply:
Changed to using fstat().